packages/client/src/components/PlayButton/button.module.css

```
@@ -0,0 +1,30 @@
+.circle,
+.button {
+  border-radius: 50%;
+}
+
+.button {
+  border: none;
+  background-color: rgb(248, 227, 86);
+  position: relative;
+  width: 150px;
+  height: 150px;
+  cursor: pointer;
+}
+
+.circle {
+  position: absolute;
+  border: 2px solid rgb(252, 127, 97);
+  top: -11px;
+  left: -1px;
+  width: 148px;
+  height: 148px;
+}
+
+.button svg {
+  fill: #fc7f61;
```

В иконке, которая в `assets/icons/play.svg` свойство цвета задано как `fill="currentColor"`, что позволяет в `css` передавать цвета как обычному тексту. 
Когда импортируешь её оттуда, можно будет использовать в таком виде:

```suggestion
  color: #fc7f61;
```